ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -37,6 +37,17 @@ def clip_by_value(t, clip_value_min, clip_value_max):
     return ivy.clip(t, clip_value_min, clip_value_max)
 
 
+@to_ivy_arrays_and_back
```

Comment:
E   tensorflow.python.framework.errors_impl.InvalidArgumentError: Value for attr 'T' of int64 is not in the list of allowed values: bfloat16, half, float, double, complex64, complex128
E   	; NodeDef: {{node Sqrt}}; Op<name=Sqrt; signature=x:T -> y:T; attr=T:type,allowed=[DT_BFLOAT16, DT_HALF, DT_FLOAT, DT_DOUBLE, DT_COMPLEX64, DT_COMPLEX128]> [Op:Sqrt]

Reply:
If I understand correctly, there are some datatypes that should not be supported by this function?

How do you check this locally? My tests didn't find this error

Reply:
Just found a more significant error: 
E   AssertionError:  the results from backend tensorflow and ground truth framework tensorflow do not match
E    [-0.]!=[-1.9073486e-06] 
E   
E   
E   Falsifying example: test_tensorflow_clip_by_global_norm(
E       input_and_norm=(['float32', 'float32'],
E        [array([-1.], dtype=float32), array([-0.5], dtype=float32)],
E        ['float32'],
E        [array([1.1754944e-38], dtype=float32)],
E        ['float32'],
E        [array([3.081488e-33], dtype=float32)]),
E       fn_tree='ivy.functional.frontends.tensorflow.clip_by_global_norm',
E       test_flags=FrontendFunctionTestFlags(
E           num_positional_args=0,
E           with_out=False,
E           inplace=False,
E           as_variable=[False],
E           native_arrays=[False],
E           generate_frontend_arrays=False,
E       ),
E       frontend='tensorflow',
E       on_device='cpu',
E   )
E   
E   You can reproduce this example by temporarily adding @reproduce_failure('6.70.1', b'AXicY2QAA0YGBM14gAEOjl+AM49tQFYKAFaoA9M=') as a decorator on your test case